packages/application-generic/src/services/in-memory-provider/index.ts

```diff
@@ -1 +1,2 @@
 export * from './in-memory-provider.service';
+export * from './redis-provider';
```

Comment:
What do you need to export the provider here for? The intention of how is set up is to isolate the providers used and not expose them to the rest of the system.

Reply:
it looks like something was exported from that module that I used in another place, but now not used anymore... will remove that line